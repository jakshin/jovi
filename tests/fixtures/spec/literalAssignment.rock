Put 123 into X
Put "Hello San Francisco" into the message

Say X
Say the message
