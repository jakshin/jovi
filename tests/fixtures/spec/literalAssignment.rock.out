123
Hello San Francisco
